test/mpi/rma/get_acc_flush.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <unistd.h>
+#include <mpi.h>
+#include "mpitest.h"
+
+#define GET_BUF_SIZE 2147467264LL
+#define GET_PER_SIZE 16384
+#define ACC_BUF_SIZE 32768
+#define SIGNATURE 0x77777777
+#define ITERATION 10
+
+/* This program tests the rma issue counter bug located at MPIDI_OFI_win_do_progress function.
+ * It needs at least 4 processes launched in round-robin way on two nodes; for example, P0 and P2
+ * are on node0, and P1 and P3 should be on node1.
+ */
+
+int main(int argc, char *argv[])
+{
+    int rank, psize, target;
+    char *win_ptr, *acc_buf, *get_buf;
+    uint32_t *signature;
+    MPI_Win win;
+    int get_cnt = 5;
+    int acc_cnt = 3;
+    int errs = 0;
+
+    MTest_Init(&argc, &argv);
+    MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+    MPI_Comm_size(MPI_COMM_WORLD, &psize);
+    assert(psize >= 3);
+
+    MPI_Win_allocate(GET_BUF_SIZE, 1, MPI_INFO_NULL, MPI_COMM_WORLD, (void **) &win_ptr, &win);
+    memset(win_ptr, 0, GET_BUF_SIZE);
+
+    if (rank != 0) {
+        acc_buf = (char *) malloc(ACC_BUF_SIZE);
+        memset(acc_buf, 0x01, ACC_BUF_SIZE);
+        target = 0;
+    } else {
+        get_buf = (char *) malloc(GET_BUF_SIZE);
+        memset(get_buf, 0, GET_BUF_SIZE);
+        signature = (uint32_t *) (get_buf + GET_BUF_SIZE - sizeof(uint32_t));
+        *signature = SIGNATURE;
+        target = 1;
```

Comment:
I understand that checking all elements in such large get buffer might be unnecessary. But can we add signature at both buffer begin and buffer end to make the check stricter?

Reply:
May I know why you think checking the beginning of the buffer is necessary?

Reply:
I misunderstood the issue. So the issue is just that the last get is not complete, right? Please ignore this comment

Reply:
yes